### PR TITLE
Tests added for email_nag / Couple of global variables changed into l…

### PR DESCRIPTION
**Pull request by anoopvalluthadam**
…ocal / Deleted unused variables

### CODE REVIEW

**Comment by anoopvalluthadam** [comment]
@sylvestre all the file changes depends the email_nag testing. That's why I initiated all changes in a single Pull request.

File: auto_nag/scripts/email_nag.py [hunk 1]
```diff
@@ -217,6 +218,11 @@ def sendMail(toaddrs, msg, username, password, dryrun=False):
 
     options, args = parser.parse_known_args()
 
+    if options.dryrun:
```
Comment (sylvestre) [inline auto_nag/scripts/email_nag.py]: I think you can simplify these lines with a single line : phonebook.PhonebookDirectory(options.dryrun)

File: auto_nag/scripts/phonebook.py [hunk 2]
```diff
@@ -35,11 +32,19 @@
...
 
 class PhonebookDirectory():
-
-    def __init__(self, config=CONFIG_JSON):
-        config = json.load(open(config, 'r'))
+    def __init__(self, dryrun=False):
         print "Fetching people from phonebook..."
-        self.people = json.loads(requests.get(PEOPLE_URL, auth=(config['ldap_username'], config['ldap_password'])).content)
+        if dryrun:
+            people_json = (get_project_root_path()
```
Comment (sylvestre) [inline auto_nag/scripts/phonebook.py]: Could you move the people_json = people_json_file out of the this conditionnal. We would have a single declaration to load the json

**Comment by sylvestre** [comment]
Besides the two minor comments, it looks great :)

File: auto_nag/scripts/phonebook.py (hunk 2 above)
Comment (anoopvalluthadam) [inline auto_nag/scripts/phonebook.py]: Sorry, I didn't get you!

Comment (sylvestre) [inline auto_nag/scripts/phonebook.py]: we would have if dryrun:
  foo
else:
   bar
self.people = json.loads(the_json_file)

Comment (anoopvalluthadam) [inline auto_nag/scripts/phonebook.py]: Understood :+1:

**Comment by anoopvalluthadam** [comment]
@sylvestre Please let me know is there anything else you want me to change.

**Pull request merged**